### ISSUE TITLE
EF3 and USB

### ISSUE THREAD

**Issue by uffdaken** [GideonZ/1541ultimate#267]
Hi there, if I have a EF3 cartridge plugged in, and startup Super Snapshot v5, then if I try to load a program, say Turbo Macro Pro., the USB0 hangs on "Searching for .......". I guess this is the normal way to load with a real C64. But I am having to load TMP first and then startup SSv5 from EF3. I use the EF3 cartridge because of the Reset and Special buttons.  I have been able to load up HESMON in EF3, exit and been able to load a BASIC program. So I think SSv5 is interfering with the search and loading on the USB ports.

**Comment by GideonZ** [comment]
What machine are you using?
What firmware are you using?
What do you mean by loading from USB, which interface are you using?

**Comment by uffdaken** [comment]
Using U64 v1.41 - 3.10. I tried to load (3 different methods) from the (1) BASIC screen - Load "tmp+reu",8,1 and then tried (2) mount in U64 directory then RUN from the BASIC screen and then (3) Mount and Run from the U64 Directory.  Was using USB0 or USB1 ports. I have attached the SSv5 bin that I installed in the SSv5 Freezer slot in EF3. Also note I tried this with the EF3 cartridge in the Expansion port by itself. But normally use a Retro Innovation EX-Pander 3. EF3 is in the first port. Have a Comal 80 (modern components) cartridge in port 2. Have no power to Port 2 or 3. Take Care. P.S  I should be getting my C64C back soon and also getting a C-128. I plan to do some experiments with the C64C and U64 (have two User Port adapters) using a Cardco G+ Parallel Printer connector  with Retro Printer w/ Raspberry Pi 3b+ to see if I can print from my HP Envy 5330 USB Printer.

Ken Simmons
